app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -94,11 +93,11 @@ class AppointAgentController @Inject()(
                     None)))
             case None =>
               val errors: List[FormError] = List(invalidAgentCode)
-              agentsConnector.ownerAgents(request.organisationId) flatMap { ownerAgents =>
+              getAgents(request.organisationId) flatMap { ownerAgents =>
```

Comment:
i think we have agreed to not use infix notation for `map` `flatMap` etc. in production code. It's ok in tests to say `foo shouldBe bar` and not `foo.shouldBe(bar)` but in code we'd like to always consistently use the dot `foo.map(_ => true)`